src/runtime-rs/crates/resource/src/network/network_with_netns.rs

```diff
@@ -120,6 +124,21 @@ impl Network for NetworkWithNetns {
         }
         Some(endpoint)
     }
+
+    async fn remove(&self, h: &dyn Hypervisor) -> Result<()> {
+        let inner = self.inner.read().await;
```

Comment:
For the netns not created by Kata, I think no need to remove them.

Reply:
fixed